libraries/mbed/targets/cmsis/TARGET_NORDIC/TARGET_MCU_NRF51822/system_nrf51822.c

```diff
@@ -79,6 +83,17 @@ void SystemInit(void)
     while (NRF_CLOCK->EVENTS_LFCLKSTARTED == 0) {
         // Do nothing.
     }
+	
+	// Default SWIO setting, pull SWIO(p19) to low for turning antenna switch to BLE radiated path.
+#ifdef TARGET_DELTA_DFCM_NNN40
+	NRF_GPIO->PIN_CNF[19] = (GPIO_PIN_CNF_SENSE_Disabled << GPIO_PIN_CNF_SENSE_Pos)
+                                        | (GPIO_PIN_CNF_DRIVE_S0S1 << GPIO_PIN_CNF_DRIVE_Pos)
+                                        | (GPIO_PIN_CNF_PULL_Disabled << GPIO_PIN_CNF_PULL_Pos)
+                                        | (GPIO_PIN_CNF_INPUT_Disconnect << GPIO_PIN_CNF_INPUT_Pos)
+                                        | (GPIO_PIN_CNF_DIR_Output << GPIO_PIN_CNF_DIR_Pos);
+
+	NRF_GPIO->OUTCLR 		= 	(GPIO_OUTCLR_PIN19_Clear << GPIO_OUTCLR_PIN19_Pos);
```

Comment:
In the DELTA DFCM_NNN40 module, the pin 19 is connected a RF switch, that controls antenna to radiate Bluetooth 4.0 or WiFi. Pull low to select Bluetooth 4.0 and Pull high to select WiFi.
We define the default setting is pull low to select Bluetooth 4.0.